modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -93,6 +115,265 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test CanCreateEndpointWithSuccessAlert {
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
+
+		task ("When I enter a path and Scope to create the API endpoint") {
+			APIBuilderUI.createAPIEndpoint(
+				description = "Description default",
+				path = "testendpoint2",
+				retrieveType = "Collection",
+				scope = "Company");
+		}
+
+		task ("Then I can see success alert") {
+			Alert.viewSuccessMessage();
+		}
+
+		task ("And Then I can see info of created endpoint with 'Get' method, scope, path") {
+			AssertElementPresent(
+				fieldLabel = "Method",
+				locator1 = "FormViewBuilder#FIELD_LABEL");
+
+			AssertElementPresent(
+				key_value = "GET",
+				locator1 = "APIBuilder#DROPDOWN_TOGGLE");
+
+			AssertElementPresent(
+				fieldLabel = "Scope",
+				locator1 = "FormViewBuilder#FIELD_LABEL");
+
+			AssertElementPresent(
+				fieldLabel = "Path",
+				locator1 = "FormViewBuilder#FIELD_LABEL");
+		}
+	}
+
+	@priority = 4
+	test CanSeeAllFieldsInCreationModal {
+		property portal.acceptance = "true";
+
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
+
+		task ("When I click 'Add API Endpoint' button") {
+			Click(locator1 = "APIBuilder#ADD_API_ENDPOINT");
+		}
+
+		task ("Then fields Scope *, Path * and Description are present") {
+			AssertElementPresent(
+				fieldLabel = "Scope",
+				locator1 = "FormViewBuilder#FIELD_LABEL_REQUIRED");
+
+			AssertElementPresent(
+				fieldLabel = "Path",
+				locator1 = "FormViewBuilder#FIELD_LABEL_REQUIRED");
+
+			AssertElementPresent(
+				fieldLabel = "Description",
+				locator1 = "FormViewBuilder#FIELD_FROM_FIELDSET");
+		}
+	}
+
+	@priority = 4
+	test CanSeeInfoOfCreatedEndpointWithDescription {
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
```

Comment:
Could you check all the steps for this test? Thanks! there are some steps missing.

Reply:
Thanks @pedroamarall . We understand now, but normally we don't change the GWT in headless qa team. And if there is a chance we need to update the GWT, please let us know about this change, then we headless qa will update our test scenarios accordingly. 
For this case, it is ok to use the steps you are using, we will update our document, thanks

Reply:
Ok @yanan thanks for that, the next time I will talk with the team before to change